### ISSUE TITLE
How to support transparent background?

### ISSUE THREAD

**Issue by dengyuhan** [square/gifencoder#3]
Two transparent PNG synthesis becomes black

**Comment by JakeWharton** [comment]
Transparency is not supported at this time and we have no plans on adding it. Someone can contribute it if they'd like.

**Comment by dengyuhan** [comment]
If I change the Color class to ARGB, is it transparent?

**Comment by JakeWharton** [comment]
No. It requires supporting transparency throughout the whole encoding process.

**Comment by ToxicMushroom** [comment]
I've got some of it working in my fork but I'm having weird bleeding effects that change when I change the imageoptions

**Comment by ToxicMushroom** [comment]
the transparancy color is likely being touched by the ditherer and it's messing it up 
I'll see if I can fix this

**Comment by amarkovits** [comment]
any update on this?

**Comment by ToxicMushroom** [comment]
@amarkovits look at my fork, it supports transparency if you need it.
Idk if a pr would ever make it since I changed quite a lot of code. I'm also not sure that this is still being maintained.

**Comment by zeng1990java** [comment]
> @amarkovits look at my fork, it supports transparency if you need it.
> Idk if a pr would ever make it since I changed quite a lot of code. I'm also not sure that this is still being maintained.

How can i get you code. I need that feature.

**Comment by ToxicMushroom** [comment]
> 
> 
> > @amarkovits look at my fork, it supports transparency if you need it.
> > Idk if a pr would ever make it since I changed quite a lot of code. I'm also not sure that this is still being maintained.
> 
> How can i get you code. I need that feature.
https://github.com/Melijn/gifencoder @zeng1990java

**Comment by jonatino** [comment]
@ToxicMushroom can you make a PR for this? Your jitpack build on your repo is broken and theres no way to use your changes.

**Comment by Landen13** [comment]
@ToxicMushroom hey man, please, create a PR
